code42_connector.py

```diff
@@ -154,6 +181,48 @@ def _handle_list_high_risk_employees(self, param, action_result):
         action_result.update_summary({"total_count": total_count})
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    @action_handler_for("get_highrisk_employee")
+    def _handle_get_high_risk_employee(self, param, action_result):
+        username = param["username"]
+        user_id = self._get_user_id(username)
+
+        try:
+            response = self._client.detectionlists.high_risk_employee.get(user_id)
+            action_result.add_data(response.data)
+            action_result.update_summary({"is_high_risk_employee": True})
+        except Py42NotFoundError:
+            action_result.update_summary({"is_high_risk_employee": False})
+
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    @action_handler_for("add_highrisk_tag")
+    def _handle_add_high_risk_tag(self, param, action_result):
+        username = param["username"]
+        risk_tag = param["risk_tag"]
+        user_id = self._get_user_id(username)
+        response = self._client.detectionlists.add_user_risk_tags(user_id, risk_tag)
+        all_tags = response.data.get("riskFactors", [])
+        response["riskFactors"] = _convert_to_obj_list(all_tags, "tag")
```

Comment:
`riskFactors` gets converted to a list of objects all with a single key `tag`.
This makes it so that you can use `....*.tag` in playbooks and perform actions off every tag in the list